train.py

```diff
@@ -0,0 +1,71 @@
+from datetime import datetime
+
+from tensorflow.keras import callbacks, optimizers
+
+from load_dataset import load_partial_dataset
+
+
+def make_callbacks(params):
```

Comment:
Consider specifying explicit return and parameter types. It is applicable to the whole MR.